__tests__/src/rules/prefer-tag-over-role-test.js

```diff
@@ -0,0 +1,43 @@
+import { RuleTester } from 'eslint';
+import parserOptionsMapper from '../../__util__/parserOptionsMapper';
+import rule from '../../../src/rules/prefer-tag-over-role';
+
+const ruleTester = new RuleTester();
+
+const expectedError = (role, tag) => ({
+  message:
+    `Use <${tag}> instead of the "${role}" role to ensure accessibility across all devices.`,
```

Comment:
```suggestion
  message: `Use <${tag}> instead of the "${role}" role to ensure accessibility across all devices.`,
```
i smell in-editor prettier :-p